### PR TITLE
add Chromecast

### LINKED ISSUE #1168: Add Chromecast detection
see ua-parser/uap-core#211 for more info.

### PR DESCRIPTION
**Pull request by mimmi20**
I added the Chromecast as device, its platform and some tests.

fixes #1168.

### CODE REVIEW

File: resources/user-agents/browsers/chrome/chrome-android-28-to-42.json [hunk 1]
```diff
@@ -897,6 +897,13 @@
           "platforms": [
             "Android_6_0"
           ]
+        },
+        {
+          "match": "Mozilla/5.0 (*CrKey arm*) applewebkit*(*khtml*like*gecko*) Chrome/#MAJORVER#.*",
```
Comment (jaydiablo) [inline resources/user-agents/browsers/chrome/chrome-android-28-to-42.json]: Why was this added in the Android file rather than in the other files with the other patterns?

Comment (mimmi20) [inline resources/user-agents/browsers/chrome/chrome-android-28-to-42.json]: rebase wrong resolved

Comment (mimmi20) [inline resources/user-agents/browsers/chrome/chrome-android-28-to-42.json]: moved
